CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ Version History
 ====
     * All Version bumps are required to update this file as well!!
 ----
+* 21.4.0 Add begin_date_time and end_date_time to Cb::Models::Job
```

Comment:
If its just the addition of two new fields without any breaks I would lean towards making this 21.3.5 so we dont have to reroll new versions on internal etc..